doc-content/enterprise-only/processes/interacting-with-processes-defining-process-instances-variables-advanced-filters-proc.adoc

```diff
@@ -1,7 +1,7 @@
-[id='interacting-with-processes-process-instances-special-filter-proc']
-= Defining special filters for process instances
+[id='interacting-with-processes-defining-process-instances-variables-advanced-filters-proc']
+= Defining process instance variables using advanced filters
```

Comment:
Define is not the right term here, I would recommend something like show or view.

Reply:
Done